### PR TITLE
[FIT-1] Fix images path

### PR DESCRIPTION
**Pull request by g2laura**
- Fixes images path for index HTML

Before:
Screenshot 2023-01-23 at 1 46 09 pm

After the fix:
Screenshot 2023-01-23 at 1 44 43 pm

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
 <body>
     <div class="topnav">
         <a href="/">
-            <img src="/Images/pngwing.com.png" class="logo" alt="logo">
+            <img src="Images/pngwing.com.png" class="logo" alt="logo">
```
Comment (alecastes28) [inline index.html]: @g2laura looking good, however, what do you think if we change it to `"images/logo.png"`?

**Review by alecastes28 — CHANGES_REQUESTED** [review]
The feature looks good, added a non blocking comment

File: index.html (hunk 1 above)
Comment (g2laura) [reply index.html]: @alecastes28 thanks for reviewing, ok I'll apply the change

Comment (g2laura) [reply index.html]: @alecastes28 fix applied, I also removed some old images

Comment (alecastes28) [reply index.html]: @g2laura  nice work

**Review by alecastes28 — APPROVED** [review]

**Pull request merged**